src/client/components/ui/Card/Link/LinkCard.tsx

```diff
@@ -17,30 +16,33 @@ interface LinkCardProps extends Pick<React.HTMLAttributes<unknown>, 'className'>
 
 export function LinkCard({ children, className, imageUrl, link, linkLabel, title, titleAs }: React.PropsWithChildren<LinkCardProps>)  {
 
-	function LinkCardTitle({ children, className }: { titleAs?: HtmlHeadingTag } & React.HTMLAttributes<HTMLTitleElement>) {
-		return React.createElement(titleAs || 'h3', { className: className }, children);
+	function LinkCardTitle({ children, className, id }: { titleAs?: HtmlHeadingTag, id: string } & React.HTMLAttributes<HTMLTitleElement>) {
```

Comment:
```suggestion
	function LinkCardTitle({ children, className, id }: { titleAs?: HtmlHeadingTag, id: string } & React.HTMLAttributes<HTMLHeadingElement>) {
```
Parce que sinon, c'est pour la balise `title`. Et du coup, t'as pas besoin de spécifier l'`id` ou le `className` puisque ça fait partie des props d'un `hx` donc
```suggestion
	function LinkCardTitle({ children }: { titleAs?: HtmlHeadingTag } & React.HTMLAttributes<HTMLHeadingElement>) {
```